src/components/TagInput/stylesBuilder.ts

```diff
@@ -0,0 +1,77 @@
+import { TagInputConfig, defaultConfig } from './TagInput.styles';
+
+import { mergeConfigWithCustom } from '@/services/mergeConfigWithCustom/mergeConfigWithCutom';
```

Comment:
something is wrong here

Reply:
Do you mean this line or this file? I resolved errors from this file which were during a build

Reply:
When I fixed other bugs this bug appeared. I have no idea why I didn't have it before - I fixed this import